openid-sharedsignals-framework-1_0.md

```diff
@@ -800,10 +800,25 @@ events_delivered
 
 delivery
 
-> **Receiver-Supplied**, A JSON object containing a set of name/value pairs
-  specifying configuration parameters for the SET delivery method.  The actual
-  delivery method is identified by the special key "method" with the value being
-  a URI as defined in {{delivery-meta}}.
+> A JSON object containing a set of name/value pairs specifying configuration
+  parameters for the SET delivery method. The actual delivery method is
+  identified by the special key "method" with the value being a URI as defined
+  in {{delivery-meta}}. The value of the "delivery" field contains two
+  sub-fields: 
+  
+>   method
+
+> > **Receiver-Supplied**, the specific delivery method to be used. This can be
+    any one of "urn:ietf:rfc:8935" (push) or "urn:ietf:rfc:8936" (poll), but
+    not both.
```

Comment:
The method field is `String`, not sure how it's possible to specify them both :)

If this specification wants to be flexible then it might make sense to allow defining other values (delivery methods).

Reply:
Does not seem ambiguous. Please reopen issue if you think this is not resolved.